store-implementation/accumulo-store/src/main/java/uk/gov/gchq/gaffer/accumulostore/retriever/impl/AccumuloElementsRetriever.java

```diff
@@ -31,8 +31,8 @@ public AccumuloElementsRetriever(final AccumuloStore store,
                                      final User user)
             throws IteratorSettingException, StoreException {
         super(store, operation, user,
-                // includeMatchedVertex if input only contains EntityIds
-                StreamSupport.stream(operation.getInput().spliterator(), false).noneMatch(input -> EdgeId.class.isInstance(input)),
+                // includeMatchedVertex if any input are EntityIds
+                StreamSupport.stream(operation.getInput().spliterator(), false).anyMatch(EntityId.class::isInstance),
```

Comment:
Is this related to the issue or just a general improvment?

Reply:
Due to adding proper testing for matchedVertex, I realised that this more accurately recreates the Gaffer 1 behaviour.